conda/models/channel.py

```diff
@@ -32,6 +33,18 @@ def join_url(*args):
     return '/'.join(args) + '/'
 
 
+def get_default_channels_canonical_name(platform):
```

Comment:
This will need to change with #3550.  It's a bit different.  Will probably wait to merge this until #3550 is merged so we can test against it.